### DISCUSSION TITLE
Trouble pulling from Git in RStudio

### DISCUSSION THREAD

**Discussion by emmats** [atsa-es/fish550-2023#11] [General]
I'm sure there is an obvious answer here, but I am new to using Git in RStudio. I am unable to pull because I have not committed my changes to the lab 1 doc. Any tips?
>>> /usr/bin/git pull
error: Your local changes to the following files would be overwritten by merge:
	Lab-1/Lab1-ARIMA.html
Please commit your changes or stash them before you merge.
Aborting

**Comment by eeholmes** [comment]
Hi, Open the Git tab, roll over Lab1-ARIMA.html, and click Revert. Or delete that .html file. 

Then do a pull. I put that file in the .gitignore file so it shouldn't be an issue going forward.
